Releasenotes 4.4.0 (#1043)
nunit/docs: docs/articles/nunit/release-notes/framework.md

# Framework Release

## NUnit 4.4.0 - May 28, 2025

> This release is a bugfix and smaller enhancements release, with 19 issues fixed.

Reviewer: ```suggestion
This release contains bug fixes and smaller enhancements, with 19 issues fixed.
```
